app/bundles/FormBundle/Form/Type/FieldType.php

```diff
@@ -420,10 +420,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         );
 
         // Put properties last so that the other values are available to form events
+        $propertiesData = (isset($options['data']['properties'])) ? $options['data']['properties'] : [];
         if (!empty($options['customParameters'])) {
+            $formTypeOptions = array_merge($formTypeOptions, ['data'=>$propertiesData]);
```

Comment:
Can you add space like `'data' => $propertiesData` please? Should be checked automatically with Travis, not sure why it passed.